include/range/v3/view/drop_last.hpp

```diff
@@ -0,0 +1,343 @@
+/// \file
+// Range v3 library
+//
+//  Copyright Andrey Diduh 2019
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+//
+
+#ifndef RANGES_V3_VIEW_DROP_LAST_HPP
+#define RANGES_V3_VIEW_DROP_LAST_HPP
+
+#include <type_traits>
+#include <meta/meta.hpp>
+#include <range/v3/range_concepts.hpp>
+#include <range/v3/view/all.hpp>
+#include <range/v3/view/view.hpp>
+#include <range/v3/view_interface.hpp>
+#include <range/v3/view_adaptor.hpp>
+#include <range/v3/utility/functional.hpp>
+#include <range/v3/utility/counted_iterator.hpp>
+
+namespace ranges
+{
+    inline namespace v3
+    {
+        namespace aux{ namespace drop_last_view {
```

Comment:
Every `{` on its own line, every namespace causes a 4-space indent, please use `detail` instead of `aux`, and wrap all internal details in `/// \cond` / `/// \endcond` to hide them from Doxygen. Here and elsewhere. Otherwise, looks good.